java/surface-placement/README.md

```diff
@@ -2,7 +2,7 @@
 
 Position graphics relative to a surface using different surface placement modes.
 
-![Surface placement app](surface-placement.png)
+![Image of surface placement](surface-placement.png)
 
 ## Use case
 
```

Comment:
I know we're not supposed to comment on unchanged things and it won't let me, but the "How to use the sample" section indicates that there is a toggle to change the draped mode, which I don't see in the sample.